provider/policy/resource/security.py

```diff
@@ -1,40 +1,399 @@
+        "iotanalytics.amazonaws.com": {
+            "type": "aws_iot_analytics",
+            "name": "IoT Analytics",
+        },
+        "securityhub.amazonaws.com": {
+            "type": "aws_securityhub_account",
+            "name": "Security Hub",
+        },
+        "trustedadvisor.amazonaws.com": {
+            "type": "aws_trusted_advisor",
+            "name": "Trusted Advisor",
+        },
+        "firehose.amazonaws.com": {
+            "type": "aws_kinesis_firehose",
+            "name": "Kinesis Firehose",
+        },
+        "glue.amazonaws.com": {"type": "aws_glue", "name": "Glue"},
+        "quicksight.amazonaws.com": {"type": "aws_quicksight", "name": "QuickSight"},
+        "cloud9.amazonaws.com": {"type": "aws_cloud9", "name": "Cloud9"},
+        "organizations.amazonaws.com": {
+            "type": "aws_organizations_account",
+            "name": "Organizations",
+        },
+        "config.amazonaws.com": {"type": "aws_config", "name": "Config"},
+        "application-autoscaling.amazonaws.com": {
+            "type": "aws_auto_scaling",
+            "name": "Application Autoscaling",
+        },
+        "autoscaling.amazonaws.com": {
+            "type": "aws_auto_scaling",
+            "name": "Autoscaling",
+        },
+        "backup.amazonaws.com": {"type": "aws_backup", "name": "Backup"},
+        "cloudtrail.amazonaws.com": {"type": "aws_cloudtrail", "name": "Cloudtrail"},
+        "cloudwatch-crossaccount.amazonaws.com": {
+            "type": "aws_cloudwatch_crossaccount",
+            "name": "Cloudwatch Crossaccount",
+        },
+        "datapipeline.amazonaws.com": {
+            "type": "aws_data_pipeline",
+            "name": "Data Pipeline",
+        },
+        "dms.amazonaws.com": {"type": "aws_dms", "name": "DMS"},
+        "dynamodb.application-autoscaling.amazonaws.com": {
+            "type": "aws_auto_scaling",
+            "name": "DynamoDB Application Autoscaling",
+        },
+        "elasticbeanstalk.amazonaws.com": {
+            "type": "aws_elastic_beanstalk_environment",
+            "name": "Elastic Beanstalk",
+        },
+        "fms.amazonaws.com": {"type": "aws_fms", "name": "Firewall Manager"},
+        "globalaccelerator.amazonaws.com": {
+            "type": "aws_global_accelerator",
+            "name": "Global Accelerator",
+        },
+        "inspector.amazonaws.com": {"type": "aws_inspector", "name": "inspector"},
+        "logger.cloudfront.amazonaws.com": {
+            "type": "aws_cloudfront_distribution",
+            "name": "CloudFront Logger",
+        },
+        "migrationhub.amazonaws.com": {
+            "type": "aws_migration_hub",
+            "name": "Migration Hub",
+        },
+        "rds.amazonaws.com": {"type": "aws_db_instance", "name": "RDS"},
+        "sns.amazonaws.com": {"type": "aws_sns_topic", "name": "SNS"},
+    }
+
+
+class IamPolicy(ResourceProvider):
+    def __init__(self, options: BaseOptions):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [One-line docstring should fit on one line with quotes (found 4) (D200)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)